e2etest/test_manager.go

```diff
@@ -227,3 +233,35 @@ func (tm *TestManager) CatchUpBTCLightClient(t *testing.T) {
 	_, err = tm.InsertBTCHeadersToBabylon(headers)
 	require.NoError(t, err)
 }
+
+func importPrivateKey(btcHandler *BitcoindTestHandler) (*btcec.PrivateKey, error) {
+	privKey, err := btcec.NewPrivateKey()
+	if err != nil {
+		return nil, err
+	}
+
+	wif, err := btcutil.NewWIF(privKey, regtestParams, true)
+	if err != nil {
+		return nil, err
+	}
+
+	descriptor := fmt.Sprintf("combo(%s)", wif.String())
+
+	// Create the JSON descriptor object.
+	descJSON, err := json.Marshal([]map[string]interface{}{
+		{
+			"desc":      descriptor,
+			"active":    true,
+			"timestamp": "now",
+			"label":     "test key",
+		},
+	})
```

Comment:
A bit of docs on the topic:
https://bips.dev/384/
https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md

Basically `combo` allows us to import a key and handle multiple types of btc scripts with a single descriptor command.

 "now" as a timestamp tells Bitcoind to start scanning from the current blockchain height

Reply:
I would add this line with brief description above the line where we create descriptor